sdk/core/core-http/test/policies/tracingPolicyTests.ts

```diff
@@ -1,7 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { assert } from "chai";
+import chai, { assert } from "chai";
+import chaiAsPromised from "chai-as-promised";
+chai.use(chaiAsPromised);
```

Comment:
Is this necessary? Why not just `await` the promise and assert on the result instead? I've found that async functions make `chai-as-promised` obsolete.

Reply:
Definitely not necessary, I like it for `assert.isRejected` instead of try...catch but in this case I ended up with no such tests. I'll remove it

Reply:
Yeah, isRejected looks much better, I do agree with you there